docs/advanced-topics/best-practices/best-practices-dockstore.rst

```diff
@@ -0,0 +1,111 @@
+Depending on how you register the workflows on Dockstore, each entry could have the other two branches and every single tag/release
+on its Versions tab. All of this clutter can be overwhelming or confusing for users to see.
+
+Likewise, we don't recommend using even a single branch to contain multiple entries. A repository should essentially represent one project, but that practice can be a grey area.
+If a lot of different workflows or tools work together or are closely related, it can be argued that they're all part of a single project.
+If that is the case, and you want to keep them tied together this way, make sure to give the descriptor files helpful file names.
+It can also help to keep them organized using different, well-named folders for each entry.
+
+One exception to this rule is when you describe the same tool or workflow with multiple descriptor languages. For example, we describe the same bamstats tool
+in our :doc:`Getting Started with CWL <../../getting-started/getting-started-with-cwl>` and :doc:`Getting Started with WDL <../../getting-started/getting-started-with-wdl>` tutorials.
+A single tool on Dockstore can have CWL and WDL descriptor files. Users will be able to see you've provided both and can easily switch between the descriptors in the UI.
+Unfortunately, you cannot have a single workflow entry on Dockstore contain multiple descriptor types. You will have to register them as separate workflows by using the Workflow Name field during registration.
+
+
+Improving Your Entries On Dockstore
+-----------------------------------
+
+Making It Easier to Find
+^^^^^^^^^^^^^^^^^^^^^^^^
+There are a few things you can do to make your tool/workflow easier to find on our site.
+
+Let's start with making your entry easier to find on our `search <https://dockstore.org/search>`_ page. One way is to add author and description metadata to your descriptor file.
+Adding an author will make it selectable on the Author facet and a description helps because the text search uses it as one of the fields to sift through.
+For more detailed information on these metadata fields, check out the following info for each language:
+
+- :doc:`WDL Best Practices <./wdl-best-practices>`
+- :doc:`CWL Best Practices <./best-practices>`
+- :doc:`Nextflow Best Practices <./nfl-best-practices>` and `Nextflow manifest documentation <https://www.nextflow.io/docs/latest/config.html#scope-manifest>`_
+
+You can also provide a description by writing a README.md file instead. If you do not provide description metadata in your descriptor, then we will try to pull the README.md file as a fallback.
+
+Another tip was already mentioned above; host your repositories on a non-personal organization. Similarly, try to use a non-personal namespace to register your Docker images.
+Doing this will group your tools/workflows together under our ``Tool: Namespace`` and ``Workflow: Organization`` facets.
```

Comment:
Drawing on recent events, it also has the advantage of allowing you to add multiple developers that can manage your content on and off Dockstore in case you get hit by a bus